apps/web/src/components/Notifications/notifications/YouReceivedTopActivityBadge.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components';
+
+import { HStack, VStack } from '~/components/core/Spacer/Stack';
+import { BaseM } from '~/components/core/Text/Text';
+import colors from '~/shared/theme/colors';
+
+export default function YouReceivedTopActivityBadge() {
+  return (
+    <StyledNotificationContent align="center" justify="space-between" gap={8}>
+      <HStack align="center" gap={8}>
+        <StyledIconWrapper align="center" justify="center">
+          <TopMemberBadge />
+        </StyledIconWrapper>
+        <StyledTextWrapper align="center" as="span" wrap="wrap">
+          <BaseM as="span">
+            You've earned a new badge as one of the top 60 active members on Gallery!
```

Comment:
let's update this to:

`You received a new badge for being amongst the top active users on Gallery this week!`

Reply:
updated!